_projects/use-coala.md

```diff
@@ -64,14 +64,14 @@ so that coala can check most file types.
 
 Find a GitHub repository that meets all these criteria:
 
-1. 100 or more [stars](https://help.github.com/articles/about-stars/),
+1. 100 or more [stars](https://help.github.com/articles/about-stars),
```

Comment:
Are these changes related to SpaceConsistency?

Reply:
No the link bear complained even after we installed the latest version

Reply:
Then I think its better to add a new commit for this change.